src/microbe_stage/editor/MicrobeEditorReportComponent.tscn

```diff
@@ -368,7 +369,12 @@ size_flags_vertical = 3
 custom_constants/margin_right = 10
 custom_constants/margin_bottom = 20
 
-[node name="VBoxContainer" type="VBoxContainer" parent="MarginContainer/HSplitContainer/PanelContainer/MarginContainer/VBoxContainer/TimelinePanel/MarginContainer/ScrollContainer/MarginContainer"]
+[node name="GlobalEventsContainer" type="VBoxContainer" parent="MarginContainer/HSplitContainer/PanelContainer/MarginContainer/VBoxContainer/TimelinePanel/MarginContainer/ScrollContainer/MarginContainer"]
+margin_right = 809.0
+margin_bottom = 202.0
+
+[node name="LocalEventsContainer" type="VBoxContainer" parent="MarginContainer/HSplitContainer/PanelContainer/MarginContainer/VBoxContainer/TimelinePanel/MarginContainer/ScrollContainer/MarginContainer"]
+visible = false
```

Comment:
Isn't the current default the local view? So shouldn't this visibility wit the global and local container be the opposite? Or am I misremembering things?

Reply:
It's true. Anyway visibility is dynamically changed in the code during init. (Still I'll change that)